app/components/layout/sidebar/multi_level_menu_component.html.erb

```diff
@@ -0,0 +1,71 @@
+<div data-controller="collapsible">
+  <button
+    type="button"
+    class="
+      flex
+      w-full
+      items-center
+      mt-1
+      p-3
+      transition-colors
+      ease-in-out
+      delay-75
+      hover:bg-slate-200
+      dark:text-slate-200
+      dark:hover:bg-slate-700
+    "
+    aria-controls="<%= "multi-level-menu-#{title}"%>"
+    data-action="collapsible#toggle"
+  >
+    <%= viral_icon(
+      name: "cog_6_tooth",
+      classes:
```

Comment:
**nitpick:** We should probably not hard code the icon as this might be used for other purposes besides settings

Reply:
You mean you don't want the same icon for all of them?? Updated in 006d96d5658b2d4f6b9eda2537fa32e2eea59dbe